test/integration/dialects/postgres/dao.test.js

```diff
@@ -301,6 +305,27 @@ if (dialect.match(/^postgres/)) {
 
     });
 
+    describe('range', function() {
+      it('should tell me that a column is range and not USER-DEFINED', function() {
+        return this.sequelize.queryInterface.describeTable('Users').then(function(table) {
+          expect(table.course_period.type).to.equal('TSTZRANGE');
+          expect(table.available_amount.type).to.equal('INT4RANGE');
+        });
+      });
+
+      it('should stringify range with insert', function() {
```

Comment:
Especially since `on('sql')` is being removed.
Unit tests for stringify/parse and one or two integration tests should definitely be sufficient.